install.rb

```diff
@@ -260,15 +260,12 @@ def prepare_installation
     $operatingsystem = Facter.value :operatingsystem
   end
 
+  # populates constants used to specify default Windows directories
+  require 'win32/dir' if $operatingsystem == "windows"
```

Comment:
Is it reasonable to assume that if we don't have win32/dir, we also don't have the others the message mentions? Or should I make it more specific?